.github/workflows/test.yml

```diff
@@ -11,10 +11,12 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v1
-    - uses: cachix/install-nix-action@releases/v7
-    - uses: cachix/cachix-action@v4
+    - uses: cachix/install-nix-action@releases/v9
+      with:
+        skip_adding_nixpkgs_channel: true
+    - uses: cachix/cachix-action@v6
       with:
         name: earnestresearch-public
         signingKey: '${{ secrets.EARNESTRESEARCH_PUBLIC_CACHIX_SIGNING_KEY }}'
-        file: ci.nix
         nixBuildArgs: --argstr ghc ${{ matrix.ghc }}
```

Comment:
i think you need to pass the build args to the line below

Reply:
Yep, I think that's it.